src/ponyfills/objects.js

```diff
@@ -156,9 +161,49 @@ export function prune(map, active) {
       }
     }
   })
+
+  reattributePredicatedValues(pruned, active);
   return pruned;
 }
 
+/**
+ * @param {object} pruned
+ * @param {Set<string>} active
+ * @returns void
+ */
+export function reattributePredicatedValues(pruned, active) {
+  function reattribute(obj, _active, collected) {
+    if (typeof obj !== 'object' || obj === null) {
+      return obj;
+    }
+
+    if (obj._predicated_values) {
+      const predicatedKeyPrefix = collected.join('.');
+
+      for (let i = 0; i < obj._predicated_values.length; i++) {
+        if (_active.has(predicatedKeyPrefix + '.' + obj._predicated_values[i]._assignment_id)) {
```

Comment:
@mattstrom looking at @tinoubao PR: https://github.com/SentientTechnologies/experiments.evolv.ai/pull/601

Looks like he made is "_predicate_assignment_id" instead of just "_assignment_id" so that needs to be changed here.